### DISCUSSION TITLE
Tags

### DISCUSSION THREAD

**Discussion by vonglan** [helgoboss/helgobox#554] [Q&A] [answered]
I have a question about tags, as described in the user guide here: [https://www.helgoboss.org/projects/realearn/user-guide#_should_i_use_conditional_activation_or_the_enabledisable_mapping_target](url).

It says:
> If you use the alternative to conditional activation, [Realearn: Enable/disable mappings](https://github.com/helgoboss/realearn/discussions/new#realearn-enable-disable-mappings), you throw both of these concerns into one mapping!

I thought that I need two mappings also in the "tag" case. The thing that I don't need, the level of indirection that becomes obsolete, is the group. Correct? Or where am I wrong?

**Comment by helgoboss** [comment, marked as answer]
You are right, it's a bit confusing what I wrote there. The level of indirection that you have with conditional activation is the internal ReaLearn parameter (a kind of "mediator"). If you use "ReaLearn: Enable/disable mappings", you don't need that mediator, you change the mapping on/off state directly.
